src/shared/utils/parse-cache-filename.js

```diff
@@ -0,0 +1,71 @@
+const assert = require('assert')
+
+/**
+ * The cache filename has several significant parse which are referred
+ * to during cache load, prior to scraping.
+ */
+function parse (filename) {
+  const re = /^(\d{4}-\d{2}-\d{2}t\d{2}_\d{2}_\d{2}\.\d{3}[Zz])-([a-z]+)(-\d+)?-([a-h0-9]{5})\.([a-z]+)\..*$/
```

Comment:
This appears to assume .gz; previous versions did not, so you could inspect and load uncompressed files when working locally.

Also, should never be `Z`, only `z`; was there a capital Z anywhere? If so that is a bug.

Reply:
Updated, allows non-gz, and test added for that.

Only `z` now, thanks.